src/main/java/com/google/devtools/build/lib/server/GrpcServerImpl.java

```diff
@@ -556,13 +570,13 @@ public void run() {
     pidFileWatcherThread = new PidFileWatcherThread();
     pidFileWatcherThread.start();
     idleServerTasks = new IdleServerTasks();
```

Comment:
I think it's better not to create the IdleServerTasks if we don't have to, so perhaps rename the option and check for nullness below. WDYT?

Reply:
You mean rename the command line option or something else?

The name `IdleServerTasks` implies a generic framework for executing tasks at idle not just GC. Of course, it's not currently used for anything else. So, if I make the refactor I think you're suggesting, maybe I should just rename it `IdleGCTaskRunner`.

Reply:
My suggestion is to keep the name IdleServerTasks and the generic framework.
But instead of disabling the GC part of it with --idle_gc=no, I would disable the whole IdleServerTasks with --idle_server_tasks=no.